src/types.ts

```diff
@@ -11,14 +11,17 @@ export interface Team {
   powerRank: number;
   powerRankText: string;
   helmetImage?: string;
+  draftGrade: string;
```

Comment:
added draft grade to the Team object since it comes from yahoo's api and could be cool to put somewhere